css-four-column-layout/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  box-sizing: border-box;
+  background-color: #f7f8fa;
+  font-family: 'Barlow', sans-serif;
+}
+
+.container {
+  max-width: 1000px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.col-heading,
+p {
+  display: block;
+  text-align: center;
+  font-size: 0.9vw;
+}
+
+h2 {
+  text-align: center;
+}
+
+h3 {
+  font-weight: 900;
+}
+
+.row-items {
+  display: flex;
+}
+
+img {
+  width: 250px;
+  height: 500px;
+  object-fit: scale-down;
+  display: inline-block;
+}
+
+img.secondary-image {
+  width: 50px;
+  height: 50px;
+  display: inline-block;
+}
+
+div.secondary-image-heading {
+  display: block;
+  text-align: center;
+  margin-top: -50px;
```

Comment:
You definitely want to be avoiding negative values for margin and padding. If you feel the need for a negative margin, it is likely that there is another element you can just reduce the margin on.